lib/matplotlib/cbook.py

```diff
@@ -2385,7 +2384,7 @@ def pts_to_midstep(x, *args):
     # do normalization
     vertices = _step_validation(x, *args)
     # create the output array
-    steps = ma.zeros((vertices.shape[0], 2 * len(x)), np.float)
+    steps = np.zeros((vertices.shape[0], 2 * len(x)), np.float)
```

Comment:
The only change on this line is `ma` -> `np`, so I don't understand what you mean, unless you are referring to somewhere else.